app/src/main/java/ch/epfl/polybazaar/database/datastore/DataStore.java

```diff
@@ -14,8 +17,8 @@
      * @param documentPath document name (ID)
      * @param callback a DataSnapshotCallback interface implementation
      */
-    void fetchData(@NonNull final String collectionPath,
-                   @NonNull final String documentPath, @NonNull final DataSnapshotCallback callback);
+    void fetch(@NonNull final String collectionPath,
+               @NonNull final String documentPath, @NonNull final DataSnapshotCallback callback);
```

Comment:
This is still here because of some tests that I was not able to adapt for now. I will delete this once the tests are changed.
You are perfectly right about the name thing, it should be consistent.